### PR TITLE
flow: enable TCP metric through API instead of config file

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

File: api/capture.go [hunk 1]
```diff
@@ -40,11 +40,12 @@ type Capture struct {
 	Name           string `json:"Name,omitempty"`
 	Description    string `json:"Description,omitempty"`
 	Type           string `json:"Type,omitempty"`
-	Count          int    `json:"Count,omitempty"`
+	Count          int    `json:"Count"`
```
Comment (safchain) [inline api/capture.go]: I thought it could be useful to have always this information

**Comment by safchain** [comment]
rerun tests

**Comment by safchain** [comment]
rerun tests

**Comment by safchain** [comment]
rerun tests

**Comment by vperelmax** [comment]
I reviewed the code. The code looks fine and it is good that now we enable it via API!

File: flow/flow.go [hunk 2]
```diff
@@ -81,6 +80,14 @@ type RawPackets struct {
 	RawPackets []*RawPacket
 }
 
+// FlowOpts describes options that can be used to process flows
+type FlowOpts struct {
+	ParentUUID string
```
Comment (nplanel) [inline flow/flow.go]: why putting parentUUID / L2 and L3 as flow option ? It's not flow options
but may be factorize as FlowUUIDs struct

File: flow/table.go [hunk 3]
```diff
@@ -324,7 +325,14 @@ func (ft *Table) flowPacketToFlow(packet *Packet, parentUUID string, t int64, L2
 	key := KeyFromGoPacket(packet.gopacket, parentUUID).String()
 	flow, new := ft.getOrCreateFlow(key)
 	if new {
-		flow.Init(key, t, packet.gopacket, packet.length, ft.nodeTID, parentUUID, L2ID, L3ID)
+		opts := FlowOpts{
```
Comment (nplanel) [inline flow/table.go]: same here with line 324  flowPacketToFlow(packet *Packet, fuuid flowUUIDs) *Flow

File: flow/flow.go (hunk 2 above)
Comment (safchain) [reply flow/flow.go]: I thought them as options as not always provided but yes we can use another struct. I'll do that

**Comment by safchain** [comment]
rerun functional-tests-backend-orientdb

**Comment by lebauce** [comment]
run skydive-functional-tests-backend-orientdb

**Pull request merged**